core/internal/tensorboard/emitter.go

```diff
@@ -300,6 +297,48 @@ func (e *tfEmitter) EmitImage(
 	return nil
 }
 
+func (e *tfEmitter) verifyAndGetImagesMetadata(images []wbvalue.Image) (format string, height int, width int, err error) {
+	// Tensorboard encodes all images to png format.
```

Comment:
I think the relevant point is that they're all the *same*, not that the format is PNG, right?